### DISCUSSION TITLE
Bot should start conversation

### DISCUSSION THREAD

**Discussion by sadden3194** [chatwoot/chatwoot#5972] [Q&A]
Hi all!

It would be nice that my bot can start a conversation after the user starts the conversation.
For example:

1. User opens the web widget and clicks on the button "Unterhaltung beginnen" (=start conversation)
![Bildschirm­foto 2022-11-25 um 11 28 00](https://user-images.githubusercontent.com/17011498/203963989-6130876a-7027-4de1-85eb-eb84bb302e81.png)

2. After this my bot should send the first message instead of this empty window:
![Bildschirm­foto 2022-11-25 um 11 28 07](https://user-images.githubusercontent.com/17011498/203964252-a718b0c3-0b61-4d78-a331-036f2b8c7084.png)

At the moment, the user have to write the first message and than my bot sends the respone like this:
![Bildschirm­foto 2022-11-25 um 11 28 28](https://user-images.githubusercontent.com/17011498/203964438-c7524003-e910-4741-adf0-37b897841538.png)

The problem I see here:
The first message from the user could be very extensive (he/she describes their problem in detail).

My bot then asks: what can I do for you? Appointment or employee?

When the user selects appointment, my bot sends a link for the appointment system and the chat ends. So the user has made a problem description for nothing.

Can we handle this?

**Comment by sadden3194** [comment]
@pranavrajs 
I would be happy to receive a response

**Comment by pranavrajs** [comment]
@Ghost108 You can listen to `webwidget.triggered` event and start a conversation automatically. https://www.chatwoot.com/docs/product/others/webhook-events#webwidget_triggered

**Comment by yogesh-hack** [comment]
this is good bot, i search like this
